content/en/graphing/using_graphs/_index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Getting started with graphs
+kind: documentation
+description: Visualize your data to gain insight
+further_reading:
+- link: "https://learn.datadoghq.com/course/view.php?id=8"
+  tag: “Learning Center”
+  text: “Building Better Dashboards”
+---
+
+## The graphing editor
+
+There are two ways to interact with the Graphing Editor: using the GUI (the default method) and writing JSON (the more complete method). This page covers using the GUI. To learn more about using JSON, see [Graphing Primer using JSON][1].
```

Comment:
Not sure how i feel about describing JSON as a more "complete" method—definitely more powerful, but i feel like completeness is misleading. Could we say something like "JSON (for advanced users)"?